Scripts/AirPlayer.gd

```diff
@@ -0,0 +1,16 @@
+extends "res://Scripts/Player.gd"
+
+class_name AirPath
+
+func _ready():
+	default_speed = 165
+	default_coyote_time = INF
+
+func jump():
```

Comment:
Why are we overriding this here?  This is identical to the implementation in the Player class.